pkgs/servers/mail/dovecot/default.nix

```diff
@@ -62,6 +62,7 @@ stdenv.mkDerivation rec {
     "--with-lucene"
     "--with-icu"
   ] ++ lib.optional (stdenv.isLinux) "--with-systemdsystemunitdir=$(out)/etc/systemd/system"
+    ++ lib.optional (stdenv.isDarwin) "--enable-static"
```

Comment:
I wonder why it works for the homebrew guys. https://github.com/Homebrew/homebrew-core/blob/master/Formula/dovecot.rb#L36 what does ` --disable-dependency-tracking`?